spec/integration/profiles_spec.rb

```diff
@@ -71,6 +71,11 @@
 
     post 'Create a profile' do
       fixtures :accounts, :profiles, :rules, :hosts, :benchmarks
+
+      before do
+        accounts(:one).hosts = hosts
+      end
```

Comment:
This is the only kind of change needed in our tests. With Pundit enforcement, we must ensure the account used for testing is associated to the hosts used for testing.